src/libs/filters/focal.c

```diff
@@ -71,7 +71,7 @@ static gchar *_focal_print_func(const double value, const gboolean detailled)
 
   if(detailled)
   {
-    dt_util_dstrcat(txt, " %s", _("mm"));
+    return dt_util_dstrcat(txt, " %s", _("mm"));
```

Comment:
or probably more readable:
`txt = dt_util_dstrcat(txt, " %s", _("mm"));`

Reply:
good catch... I wonder how it could even work under linux...
thanks for the fix !

Reply:
For me both versions equally readable. And version with assignment even looks not so **logical** (and therefore less understandable). A question would arise in my head: what is purpose of unneeded assigning to variable when we will immidiately end execution flow of function and return this value (saying that I believe that modern compilers optimize this to the same code).

I just looked at how it is done in similar case in another filter (ratio.c) and did similarly.